README.md

```diff
@@ -176,12 +176,6 @@ you to manage and maintain those images and containers in a production environme
 familiar container cli commands.  For more details, see the
 [Container Tools Guide](https://github.com/containers/buildah/tree/master/docs/containertools).
 
-## Podman Legacy API (Varlink)
-Podman offers a [Varlink-based API](https://github.com/containers/podman/blob/master/docs/tutorials/varlink_remote_client.md)
-for remote management of containers. However, this API has been deprecated by the REST API.
-Varlink support is in maintenance mode, and will be removed in a future release.
-For more details, you can see [this blog](https://podman.io/blogs/2020/01/17/podman-new-api.html).
-
```

Comment:
I think it would be better to keep this, but rework it:  Perhaps

```
## Podman Legacy API (Varlink)
Podman offered a [Varlink-based API](https://github.com/containers/podman/blob/master/docs/tutorials/varlink_remote_client.md)
for remote management of containers in versions prior to Podman v3.0. However, this API has now been removed upstream and in Podman v3.0.
For more details, you can see [this blog](https://podman.io/blogs/2020/01/17/podman-new-api.html) and [this blog](https://podman.io/blogs/2020/08/01/deprecate-and-remove-varlink-notice.html).
```

Reply:
reworded.